Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -54,6 +54,7 @@ + (NSString *)longHelpText {
           @"    --block: add to block\n"
           @"    --silent-block: add to silent block\n"
           @"    --compiler: allow and mark as a compiler\n"
+          @"    --cel {cel_expr}: add a CEL rule\n"
```

Comment:
nit: we probably need to add an explanation or link to CEL documentation to the usage string so people know that how to use args, target and CEL operators.

Reply:
Done